Makefile

```diff
@@ -0,0 +1,33 @@
+npm-install:
```

Comment:
Generally speaking it's a good idea to declare these as `.PHONY` in a makefile, you can skip errors when a file called `test` is accidentally created your tests won't run.